Rakefile

```diff
@@ -263,7 +263,9 @@ namespace :package do
     s3_path = is_standalone ? Pathname.new(ZIPPED_STANDALONE) : Pathname.new(ZIPPED_BUNDLE)
     latest_obj = s3_bucket.objects[latest_path].write(s3_path)
     fastlane_obj = s3_bucket.objects[fastlane_path].write(s3_path)
+    latest_obj.content_disposition = 'attachement; filename="fastlane.zip'
```

Comment:
Ah this makes sense. I got confused by some of those docs referencing acls and we were setting it that way as well.. Ill change this now.